PocketKit/Sources/PocketKit/Refresh/HomeRefreshCoordinator.swift

```diff
@@ -1,52 +1,60 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 import Foundation
 import UIKit
 import Combine
 import Sync
+import SharedPocketKit
 
 protocol HomeRefreshCoordinatorProtocol {
+    /// Refresh method that can be called by View Model classes
+    /// - Parameters:
+    ///   - isForced: True when a user manually asked for a refresh
+    ///   - completion: The completion call back to run when the code is done running
     func refresh(isForced: Bool, _ completion: @escaping () -> Void)
 }
 
-class HomeRefreshCoordinator: HomeRefreshCoordinatorProtocol {
-    static let dateLastRefreshKey = "HomeRefreshCoordinator.dateLastRefreshKey"
-    private let notificationCenter: NotificationCenter
-    private let userDefaults: UserDefaults
+class HomeRefreshCoordinator: AbstractRefreshCoordinatorProtocol, HomeRefreshCoordinatorProtocol {
```

Comment:
what's the need of conforming to `HomeCoordinatorProtocol` since `AbstractRefreshCoordinatorProtocol` already has a `refresh(_ :, _ :)` method?